cache.rst

```diff
@@ -0,0 +1,192 @@
+                    adapter: cache.adapter.memcached
+                cache.foobar:
+                    adapter: cache.adapter.memcached
+                    provider: 'memcached://user:password@example.com'
+
+The configuration above will create 3 services: ``my_cache_pool``, ``cache.acme``
+and ``cache.foobar``.  The ``my_cache_pool`` pool is using the ArrayAdapter
+and the other two are using the :ref:`MemcachedAdapter <component-cache-memcached-adapter>`.
+The ``cache.acme`` pool is using the Memcached server on localhost and ``cache.foobar``
+is using the Memcached server at example.com.
+
+For advanced configurations it could sometimes be useful to use a pool as an adapter.
+
+.. code-block:: yaml
+
+    framework:
+        cache:
+            pools:
+                my_cache_pool:
+                    adapter: cache.adapter.memcached
+                    provider: 'memcached://user:password@example.com'
+                cache.short_cache:
+                    adapter: my_cache_pool
+                    default_lifetime: 60
+                cache.long_cache:
+                    adapter: my_cache_pool
+                    default_lifetime: 604800
+
+Creating a Cache Chain
+----------------------
+
+Different cache adapters has different strengths and weaknesses. Some might be really
+quick but small and some may be able to contain a lot of data but are quite slow.
+To get the best of both worlds you may use a chain of adapters. The idea is to
+first look at the quick adapter and then move on to slower adapters. In the worst
+case we have to recalculate the value.
+
+.. code-block:: yaml
+
+    services:
+        app.my_cache_chain_adapter:
+            class: Symfony\Component\Cache\Adapter\ChainAdapter
+            arguments:
+                - ['cache.adapter.array', 'cache.my_redis', 'cache.adapter.file']
+                - 31536000 # One year
+
+    framework:
+        cache:
+            pools:
+                my_cache_pool:
+                    adapter: app.my_cache_chain_adapter
+                cache.my_redis:
+                    adapter: cache.adapter.redis
+                    provider: 'redis://user:password@example.com'
+
+.. note::
+
+    In this configuration there is a ``cache.my_redis`` pool that is used as an
+    adapter in the ``app.my_cache_chain_adapter``
+
+
+Clearing the Cache
+------------------
+
+To clear the cache you can use the ``bin/console cache:pool:clear [pool]`` command.
+That will remove all the entries from your storage and you wil have to recalcuate
+all values. You can also group your pools into "cache clearers". There 3 cache clearer
+by default:
+
+* cache.global_clearer
+* cache.system_clearer
+* cache.app_clearer
+
+The global clearer clears all the cache in every pool. The system cache clearer
+is used in the ``bin/console cache:clear`` command. The app clearer is the default
+clearer.
+
+.. code-block:: terminal
+
+    $ bin/console cache:pool:clear cache.app_clearer
```

Comment:
We always prefix `php` explicitly: `php bin/console ...` (as that's the only way that's compatible with both *nix and Windows)